packages/compass-e2e-tests/helpers/commands/set-value-visible.ts

```diff
@@ -8,7 +9,10 @@ export async function setValueVisible(
   await browser.waitUntil(async () => {
     const element = await browser.$(selector);
     await element.waitForDisplayed();
+    await element.click(); // focus
```

Comment:
Looks like who knows how many subtle things like this broke in the webdriverio upgrade. `await element.clearValue()` now does nothing and `await element.setValue()` appends to the text that's already in the form field's value...